recipes/diligent-core/all/test_package/CMakeLists.txt

```diff
@@ -4,6 +4,21 @@ project(test_package)
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup()
 
+if(WIN32)
+  set(FILE2STRING_PATH "${CONAN_BIN_DIRS_DILIGENT-CORE}/File2String.exe")
+else()
+  set(FILE2STRING_PATH "${CONAN_BIN_DIRS_DILIGENT-CORE}/File2String")
+endif()
```

Comment:
too fragile and specific to conan world, use `find_program()` to really see that this logic works for consumers.

Reply:
It's just the test cmakelists, not a module used by consumers. I would be OK if file2string was directly called from test package's conanfile.py

Reply:
You delegate extra work to the next maintainer who will have to move this recipe to CMakeToolchain one day or another, that's all.